understanding/target-size-minimum.html

```diff
@@ -53,330 +53,238 @@
-                        the spacing between pins and neighboring pins will often be below 24 CSS pixels. It
-                        is essential to show the pins at the correct map location, therefore the Essential
-                        exception applies. A similar example is an interactive data visualization where targets
-                        are necessarily dense. Another example is where jurisdictions legally require online
-                        forms to replicate paper forms, which can impose constraints on the size of targets.
-                        In such jurisdictions, any legal requirement to replicate small targets can be considered
-                        essential. When the essential exception is applicable, authors are strongly encouraged
-                        to provide equivalent functionality through alternative means to the extent practical.
+                     <li><strong>必要不可欠:</strong> ターゲットのサイズ及び間隔が、伝達される情報にとって必須である場合、<q>必要不可欠</q>の例外が適用される。例えば、デジタル地図では、ピンの位置は地図上に表示される場所の位置に対応する。多数のピンが近接している場合、ピンと隣接するピンとの間隔は、多くの場合 24 CSS ピクセル未満になる。地図上の正しい位置にピンを表示することが必要不可欠であるため、必要不可欠の例外が適用される。同様の例としては、ターゲットが必然的に密集する対話型データ視覚化がある。別の例としては、管轄区域が紙のフォームを複製するためにオンラインフォームを法的に義務付けている場合であり、これによりターゲットのサイズに制限が課される可能性がある。そのような管轄区域では、小さなターゲットを複製するための法的要件が必要不可欠であるとみなすことができる。必要不可欠の例外が適用される場合、コンテンツ制作者は、実用的な範囲で代替手段を通じて同等の機能を提供することが強く推奨される。
+
                      </li>
                      
                   </ul>
                   
                </section>
                <section id="size-requirement">
                   
-                  <h3>Size requirement</h3>
+                  <h3>サイズ要件</h3>
                   
                   
-                  <p>For a target to be "at least 24 by 24 CSS pixels", it must be possible to draw a solid
-                     24 by 24 CSS pixel square, aligned to the horizontal and vertical axis such that the
-                     square is completely within the target (does not extend outside the target's area).
+                  <p>ターゲットを「少なくとも 24 × 24 CSS ピクセル」にするためには、正方形が完全にターゲットの内部に収まるように (ターゲットの領域外に広がらないように) 水平軸及び垂直軸に向きを揃えて、24 × 24 CSS ピクセルの実線の正方形を描画できなければならない。
                   </p>
                   
                   
                   <figure id="target-size-basic">
                      <img src="img/target-size-basic.svg" width="180" height="75" alt="Three square boxes next to each other, each with a height and width of 24px" />
                      
-                     <figcaption>Figure 1 Where targets are a 24 by 24px square (and larger is better), they meet the size requirement
-                        of the criterion and pass (image shown to scale - <a href="img/target-size-basic.svg">see the scalable original version</a>)
+                     <figcaption>図 1 ターゲットが 24 × 24 ピクセルの正方形 (大きいほど良い) の場合、達成基準のサイズ要件を満たし、合格する (画像は拡大縮小して表示している - <a href="img/target-size-basic.svg">拡大縮小可能なオリジナルバージョンを参照</a>)
```

Comment:
images shown to scale ですが、実際にページに貼り付けられている画像を見ると、24px の正方形が実寸になるように調整されているように見えます。なので、「画像は実寸表示」とするのがよいのかもしれません。

```suggestion
                     <figcaption>図 1 ターゲットが 24 × 24 ピクセルの正方形 (大きいほど良い) の場合、達成基準のサイズ要件を満たし、合格する (画像は実寸表示 - <a href="img/target-size-basic.svg">拡大縮小可能なオリジナルバージョンを参照</a>)
```

Reply:
@caztcha 

なるほど確かに。「原寸」も思い浮かびましたが、「実寸」でいきましょうか。